### DISCUSSION TITLE
Promt for path names

### DISCUSSION THREAD

**Discussion by UnknownBrain254** [iraf-community/iraf#153] [Q&A]
I'm using [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10) to install IRAF. I'm following these [instructions](https://iraf-community.github.io/install.html). I have no difficulty with downloading and unpacking the package.  The package is located

 /home/young_kindaichi

Next I have taken the following code:
```
cd  /home/young_kindaichi//iraf-2.16.1-2018.11.01 
./install 
```
Which now shows the  following window
```
                ====================================
                       IRAF v2.16 Installation
                ====================================

       Welcome to the IRAF installation script.   This script will first
  prompt you for several needed path names.   Once  the  installation is
  complete, you will be allowed to do some minimal system configuration.

  For each prompt: hit <CR> to accept the default value, 'q' to quit,
  or 'help' or '?' to print an explanation of the prompt.


========================================================================
=====================  Query for System Settings  ======================
========================================================================
```

As given in the manual that is "The script will prompt you for the path to the default image directory, the cache directory, and the binary files directory. Usually, you can everywhere use the default settings when asked from the install script". 

I used the following code:
```
New iraf root directory (/home/young_kindaichi/iraf-2.16.1-2018.11.01): <CR>
Default root image storage directory (/home/young_kindaichi/.iraf/imdir): <CR>

   ***  The definition of imdir cannot be the same as the iraf
   ***  root, please choose a different directory.  Ideally this
   ***  should be some large data area on your system or a user
   ***  data area such as /home, /users, /u1, etc.

Default root image storage directory (/home/young_kindaichi/.iraf/imdir):
```

What do I do now? Please consider that I'm a beginner to Linux and know nothing about it. I can follow what you will say. Please help me

**Comment by olebole** [comment]
You can just hit "Enter" on all questions to accept the default. Alternatively (if you installed Ubuntu in WSL), you also can install IRAF via `sudo apt install iraf`.
However, I strongly recommend to not start with IRAF. IRAF is an old, abandoned piece of software, which is kept here mainly for people who still rely on this.
Instead I recommend to use [astropy](https://astropy.org) and its affiliated packages.

**Reply by UnknownBrain254** [reply]
I used ``sudo apt install iraf`` to install iraf. And it shows no errors. I typed ``dpkg -s iraf`` to see if it's installed. And It's installed. What  I have to use it? I should type ``iraf`` to open it but then it shows 
```[2] 37
[1]   Exit 127                xgterm -e cl
young_kindaichi@LAPTOP-K91A53NB:~$
Command 'xgterm' not found, did you mean:

  command 'xterm' from deb xterm (353-1ubuntu1.20.04.2)
```


So Can you help me with this? 

I would try to use astropy but My professor asked me to install iraf first. I'll ask him if I can use astropy instead. Thank you for your consideration.

**Reply by olebole** [reply]
Maybe you tell your professor that IRAF is dead and that makes more sense to use (and improve) Astropy instead? Or ask him to help keeping IRAF installable. There is no support anymore since years.

For graphics, you need xterm or xgterm. XTerm can be installed with `apt install xterm`, and xgterm (for Ubuntu 20.11 and later) `apt install xgterm`.

**Reply by efrlux12** [reply]
> I used `sudo apt install iraf` to install iraf. And it shows no errors. I typed `dpkg -s iraf` to see if it's installed. And It's installed. What I have to use it? I should type `iraf` to open it but then it shows
> 
> ```
> [1]   Exit 127                xgterm -e cl
> young_kindaichi@LAPTOP-K91A53NB:~$
> Command 'xgterm' not found, did you mean:
> 
>   command 'xterm' from deb xterm (353-1ubuntu1.20.04.2)
> ```
> 
> So Can you help me with this?
> 
> I would try to use astropy but My professor asked me to install iraf first. I'll ask him if I can use astropy instead. Thank you for your consideration.

hello, were you able to solve your problem?
Im trying to open IRAF but from an "interface emulator" of ubuntu, but it gives me an error, specifically it tells me `command not found` when I try to open the program, so I thought that if you could solve your problem, you would help me with mine, thank you.

**Reply by olebole** [reply]
Please provide more details: 
What exactly did you do? What is the "interface emulator" you used? How did you "try to open the program"?
